evm/src/cpu/kernel/aggregator.rs

```diff
@@ -84,13 +85,12 @@ mod tests {
 
     #[test]
     fn make_kernel() {
-        let _ = env_logger::Builder::from_default_env()
-            .format_timestamp(None)
-            .try_init();
+        env_logger::init_from_env(
+            env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "debug"),
+        );
 
         // Make sure we can parse and assemble the entire kernel.
         let kernel = combined_kernel();
         debug!("Total kernel size: {} bytes", kernel.code.len());
-        dbg!("Total kernel size: {} bytes", kernel.code.len());
```

Comment:
(This syntax wasn't working; was the issue that it wasn't convenient to add `RUST_LOG=debug`? If so the change above should help.)

Reply:
Oh sorry, I forgot to remove this. I just do this sometimes because it's less keystrokes to add a `dbg!` than to set the env var, but setting it to "debug" by default also works 👍